react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import './Accordion.css';
+
+export default function Accordion({ languages }) {
+  const [clicked, setClicked] = useState('');
+
+  function handleLanguageClick(language) {
+    clicked === language ? setClicked('') : setClicked(language);
+  }
+
+  return (
+    <Languages
+      languages={languages}
+      handleOnClick={handleLanguageClick}
+      clicked={clicked}
+    />
+  );
+}
+
+function Languages({ languages, handleOnClick, clicked }) {
```

Comment:
The prop for an event handler should be named with `on` while the function that handles the event should be named with `handle`.

So this should take a prop named `onClick` and it should be passed (on line 14) a function that starts with `handle` (which you've done correctly).